docker-images/sw360empty/docker-entrypoint.sh

```diff
@@ -208,23 +227,40 @@ if [ "$FOSSOLOGY_HOST" ] && [ "$FOSSOLOGY_PORT" ]; then
 
     FOSSOLOGY_KEY_LOCATION=${FOSSOLOGY_KEY_LOCATION:-/run/secrets/fossology.id_rsa}
     if [ -f "$FOSSOLOGY_KEY_LOCATION" ]; then
-        ln -s "$FOSSOLOGY_KEY_LOCATION" /etc/sw360/fossology.id_rsa
+        cp  "$FOSSOLOGY_KEY_LOCATION" /etc/sw360/fossology.id_rsa
```

Comment:
this adds the secret to the state of the docker container.

Reply:
/etc is not part of the state AFAIK. Nevertheless I don't know why this change is there and will revert it.